src/components/login/index.js

```diff
@@ -0,0 +1,50 @@
+import React, { PropTypes, Component } from 'react'
+import { translate } from 'react-i18next'
+
+class Login extends Component {
+	constructor (props) {
+		super(props)
+		this._handleSubmit = this.handleSubmit.bind(this)
```

Comment:
Can do, but I like to keep the `render` as simple as possible.
The `handleSubmit` method is not a oneliner therefore it benefits from being outside of it.

Imho it makes the whole thing simpler to read. But if you're both against that, I can change it 👍